### PR TITLE
target/riscv: Set hypervisor bits.

### PR DESCRIPTION
**Pull request by timsifive**
No other attempt is made at doing anything hypervisor-specific. Are other things necessary?

Change-Id: Ib65f114888840cf0878f9bfe028c9a42b436aa3e

### CODE REVIEW

File: src/target/riscv/riscv.c [hunk 1]
```diff
@@ -732,16 +732,20 @@ static struct match_triggers_tdata1_fields fill_match_triggers_tdata1_fields_t2(
 static struct match_triggers_tdata1_fields fill_match_triggers_tdata1_fields_t6(
 		struct target *target, struct trigger *trigger)
 {
-	RISCV_INFO(r);
+	bool misa_s = riscv_supports_extension(target, 'S');
+	bool misa_u = riscv_supports_extension(target, 'U');
+	bool misa_v = riscv_supports_extension(target, 'V');
```
Comment (GregSavin) [inline src/target/riscv/riscv.c]: Is this intended to be checking for the hypervisor extension?

Comment (timsifive) [reply src/target/riscv/riscv.c]: Right. The virtualization extension.

Good catch. Thanks.

**Review by GregSavin — DISMISSED** [review]
Looks good to me, though I have no experience yet with bring-up (or debug) of a RISC-V hypervisor environment.  I guess the OpenOCD level of debugging is exactly what somebody would use to get that sort of thing wiggling.

**Comment by timsifive** [comment]
Now why did github not rerun the actions so I get a Windows binary?

**Comment by timsifive** [comment]
I pushed a no-op change, and now they're running...

**Review by JanMatCodasip — COMMENTED** [review]
I have little practical experience with H extension but the code change looks all right to me.

**Comment by timsifive** [comment]
I'm ignoring `checkpatch` because it's complaining about upstream changes, not changes in this branch.

**Review by GregSavin — APPROVED** [review]

**Pull request merged**